src/main/resources/application.yml

```diff
@@ -31,7 +31,7 @@ spring:
 folio:
   tenant:
     validation:
-      enabled: true
+      enabled: false
```

Comment:
Could you explain why the validation is disabled? I don't think it's correct

Reply:
Sorry, this is my fault. Changed for local testing, forgot to change back